src/components/ModalConfirmCode.tsx

```diff
@@ -0,0 +1,108 @@
+import { connected } from 'process';
+import React,  {FC, useEffect, useState} from 'react';
+import './ModalConfirmCode.scss';
+
+interface CardProps {
+  active: boolean;
+  setActive: (bool: boolean) => void ;
+}
+
+const ModalConfirmCode: FC<CardProps> = 
+  ({
+    active, 
+    setActive,
+  }) => {
+  
+  const [code, setCode] = useState<string>('');
+  const [codeDirty, setCodeDirty] = useState<boolean>(false);
+  const [codeError, setCodeError] = useState<string>('The input field must be filled');
+  const [formValid, setFormValid] = useState<boolean>(false);
+
+  useEffect(() => {
+    if (codeError) {
```

Comment:
More pretty form instad of if:

setFormValid(!codeError)

Reply:
Maby I can leave as it done?

Reply:
You can, but for me will be less inspiration to give you comments

Reply:
Ok, done